libraries/api-documenter/src/markdown/MarkdownGenerator.ts

```diff
@@ -210,45 +203,63 @@ export class Documenter {
       MarkupBuilder.createTextElements('Description')
     ]);
 
-    for (const memberName of Object.keys(docClass.members)) {
-      const member: ApiMember = docClass.members[memberName];
-      const memberNode: DocumentationNode = new DocumentationNode(member, memberName, classNode);
+    for (const docMember of docClass.children) {
+      const apiMember: ApiMember = docMember.apiItem as ApiMember;
 
-      switch (member.kind) {
+      switch (apiMember.kind) {
         case 'property':
           const propertyTitle: MarkupBasicElement[] = [
             MarkupBuilder.createApiLink(
-              [MarkupBuilder.createCode(memberName, 'javascript')],
-              memberNode.getApiReference())
+              [MarkupBuilder.createCode(docMember.name, 'javascript')],
+              docMember.getApiReference())
           ];
 
           propertiesTable.rows.push(
             MarkupBuilder.createTableRow([
               propertyTitle,
               [],
-              [MarkupBuilder.createCode(member.type, 'javascript')],
-              MarkupBuilder.renderDocElements(member.summary)
+              [MarkupBuilder.createCode(apiMember.type, 'javascript')],
+              MarkupBuilder.renderDocElements(apiMember.summary)
+            ])
+          );
+          this._writePropertyPage(docMember);
+          break;
+
+        case 'constructor':
+          // TODO: Extract constructor into its own section
+          const constructorTitle: MarkupBasicElement[] = [
+            MarkupBuilder.createApiLink(
+              [MarkupBuilder.createCode(RenderingHelpers.getConciseSignature(docMember.name, apiMember), 'javascript')],
+              docMember.getApiReference())
+          ];
+
+          methodsTable.rows.push(
+            MarkupBuilder.createTableRow([
+              constructorTitle,
+              [],
```

Comment:
>[], [](start = 14, length = 3)

do you get the accessmodifier?

Reply:
IApiConstructor currently doesn't support the access modifier.  It's one of the *.api.json improvements that will get fixed in an upcoming PR.

---
In reply to: [141506814](https://github.com/Microsoft/web-build-tools/pull/381#discussion_r141506814) [](ancestors = 141506814)